frontend/src/components/SchemaList.vue

```diff
@@ -1,23 +1,86 @@
 <template>
-  <ul class="list-group sticky-top">
-    <li
-      v-on:click="$emit('selectSchema', schema)"
-      v-for="schema in schemas"
-      :key="schema.id"
-      class="list-group-item"
-      :class="selectedSchema === schema ? 'active' : ''"
-    >
-      {{ schema.name }}
-    </li>
-  </ul>
+  <li class="nav-item dropdown">
```

Comment:
In general I like this option more than my initial when schemas list was sitting all the time to the left of entities list taking up space, but:
1. I guess we need ordering of schemas by name here (currently it displays them just as they were received)
2. Maybe we need to deal somehow with a big list of schemas. For example, if we have a list of 100 schemas it will just open this dropdown with all schemas and with vertical scroll of several display heights. But maybe that's okay🤔

Reply:
I think/hope that the number of schemas should be relatively small.

On the other hand we will also need some feature to find/access deleted schemas. I was hoping to address all of that in a second PR...